pom.xml

```diff
@@ -12,16 +12,18 @@
         <description>IUCDS pilot AWS Middleware</description>
 
         <parent>
-                <groupId>com.cooldatasoft</groupId>
```

Comment:
keep this as the parent as this pom has the plugin configured not the one you have put in